src/components/Checkbox/Checkbox.tsx

```diff
@@ -53,8 +53,8 @@ class Checkbox extends React.PureComponent<CombinedProps, never> {
 
   componentDidUpdate({disabled: prevDisabled}: Props) {
     if (
+      prevDisabled === false &&
       this.props.disabled &&
-      !prevDisabled !== true &&
```

Comment:
This makes my eyes go crossed. Good fix

Reply:
Obviously could just make this `prevDisabled`, but I'd rather avoid Type Coercion.

Reply:
Wait. I actually don't even think we need this line. 

`{disabled: prevDisabled}: Props` means we can access this with `this.props.disabled`, which is already the condition underneath.